open_humans/models.py

```diff
@@ -257,3 +257,24 @@ def create(cls, rss_feed_entry):
     @property
     def published_day(self):
         return arrow.get(self.published).format('ddd, MMM D YYYY')
+
+class GrantProject(models.Model):
+    """
+    Store data about an ongoing grant project.
+    """
+    name = models.CharField(max_length=255, unique=True)
+    grant_date = models.DateField(null=True)
+    status = models.CharField(max_length=120)
+    github = models.TextField(blank=True)
+    grantee_name = models.CharField(max_length=255)
+    photo = models.ImageField(
+        blank=True, 
+        max_length=1024,
+        # Stored on S3 TODO: update these fields
```

Comment:
Is this still a todo? I've no idea how the S3 uploads currently work for other photos.

Reply:
I copied the image storage method from other models and left that TODO for myself. I believe this will work properly, but I am not sure how the uploads work either so maybe that would be good to understand before I make changes (or not) to the photo bit

Reply:
I'm comfortable with treating it as if it works, and if it doesn't, fixing it later. But maybe removing the TODO? :)